V Up command broken

**V version:** master
**OS:** Android (Termux)

**What did you do?**
Ran `v up`

**What did you expect to see?**
V updated successfully
 
**What did you see instead?**
```
$ v up                                                
Updating V...                                         
From https://github.com/vlang/v                      
 * branch              master     -> FETCH_HEAD          
cedf185b..784d27f4  master     -> origin/master    
Updating cedf185b..784d27f4                           
Fast-forward                                           
examples/tcp_echo_server.v | 35 +++++++++++++++++++++++++++++++++++                                         
1 file changed, 35 insertions(+)                      
create mode 100644 examples/tcp_echo_server.v        
Current branch master is up to date.                  
cd /data/data/com.termux/files/usr/tmp/vc && git clean -xf && git pull --quiet                         
make fresh_tcc                                       
make[1]: Entering directory '/data/data/com.termux/files/home/v'                                         
make[1]: Nothing to be done for 'fresh_tcc'.          
make[1]: Leaving directory '/data/data/com.termux/files/home/v'                                             
cc  -std=gnu11 -w -o v /data/data/com.termux/files/usr/tmp/vc/v.c  -lm                            
/data/data/com.termux/files/usr/tmp/vc/v.c:563:10: fatal error: 'sys/termios.h' file not found
#include <sys/termios.h> // TIOCGWINSZ                         
^~~~~~~~~~~~~~~                              
1 error generated.                                    
make: *** [Makefile:51: all] Error 1
```

--- a/vlib/term/term_nix.v
+++ b/vlib/term/term_nix.v
@@ -4,11 +4,6 @@ import os
 
 #include <sys/ioctl.h>
 
-$if solaris {
-#include <unistd.h> // ioctl
-#include <sys/termios.h> // TIOCGWINSZ
-}
-
 pub struct C.winsize {
 pub:
 	ws_row    u16

--- a/vlib/term/term_solaris.v
+++ b/vlib/term/term_solaris.v
@@ -1,0 +1,5 @@
+module term
+
+#include <unistd.h> // ioctl
+
+#include <sys/termios.h> // TIOCGWINSZ

